pkg/app/controller.go

```diff
@@ -297,6 +298,21 @@ func logPolicyMode(mode sysconf.PolicyMode, logger *zap.Logger) {
 	}
 }
 
+func saveConfigPatches(patches []block.Patch, store *confmerge.DirStore, logger *zap.Logger) {
+	if len(patches) == 0 {
+		return
+	}
+	ref, err := store.SavePatches(patches)
```

Comment:
I'm not sure SavePatches deserves to be a method on store anymore considering it's not used by the confmerge package at all.